src/components/Navbar.js

```diff
@@ -113,13 +113,13 @@ class BounceNavbar extends Component {
     }
 
     search() {
-        this.props.client.searchClubs(this.props.searchQuery)
+        this.props.client.searchClubs({name:this.props.searchQuery, description:this.props.searchQuery})
```

Comment:
Is this change supposed to be here? I think just passing the search query is okay as that is what the searchClubs GET endpoint expects.